examples/custom-prefetch-methods/custom_prefetch_methods.bal

```diff
@@ -0,0 +1,92 @@
+import ballerina/graphql;
+import ballerina/graphql.dataloader;
+import ballerina/http;
+
+public type Book record {|
+    int id;
+    string title;
+|};
+
+type BookRow record {|
+    readonly int id;
+    string title;
+    int author;
+|};
+
+type AuthorRow record {|
+    readonly int id;
+    string name;
+|};
+
+final readonly & table<AuthorRow> key(id) authorTable = table [
+    {id: 1, name: "J.K. Rowling"},
+    {id: 2, name: "Stephen King"},
+    {id: 3, name: "Agatha Christie"},
+    {id: 4, name: "Haruki Murakami"},
+    {id: 5, name: "Jane Austen"}
+];
+
+final readonly & table<BookRow> key(id) bookTable = table [
+    {id: 1, title: "Harry Potter and the Sorcerer's Stone", author: 1},
+    {id: 2, title: "The Shining", author: 2},
+    {id: 3, title: "Murder on the Orient Express", author: 3},
+    {id: 4, title: "Norwegian Wood", author: 4},
+    {id: 5, title: "Pride and Prejudice", author: 5},
+    {id: 6, title: "Harry Potter and the Chamber of Secrets", author: 1},
+    {id: 7, title: "It", author: 2},
+    {id: 8, title: "And Then There Were None", author: 3},
+    {id: 9, title: "Kafka on the Shore", author: 4}
+];
+
+isolated function bookLoaderFunction(readonly & anydata[] ids) returns BookRow[][]|error {
```

Comment:
We can't do this since the BatchLoadFunction type expects `readonly & anydata[]` as argument. We can only override the return type not the parameter type